lte/gateway/python/magma/mobilityd/tests/rpc_servicer_tests.py

```diff
@@ -329,6 +330,30 @@ def test_get_ip_for_unknown_subscriber(self):
         self.assertEqual(err.exception.code(),
                          grpc.StatusCode.NOT_FOUND)
 
+    def test_get_set_gw_info(self):
```

Comment:
why "set" is in the function name? Looks like we are getting the gateway info from mobilityd and from shell, and compare....

Reply:
fixed